### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Mcfreestyle**
Some added lines

### CODE REVIEW

File: hello.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env python3
 print ("hello world")
-print("HOLA")
+print("HOLA")
```
Comment (Chikopes) [inline hello.py]: You can use a only language, spanish or english

File: index.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<html>
+	<head>
+		<title>One more time in collaboration</title>
```
Comment (Chikopes) [inline index.html:3]: Add more elements in the header

File: login.php [hunk 3]
```diff
@@ -1,3 +1,4 @@
 <?php
 $usuario;
-$password;
\ No newline at end of file
+$password;
+?>
```
Comment (Chikopes) [inline login.php:4]: Its OK

**Review by Chikopes — COMMENTED** [review]
corrects the suggestions please

File: hello.py (hunk 1 above)
Comment (Mcfreestyle) [reply hello.py]: it's in english now

File: index.html (hunk 2 above)
Comment (Mcfreestyle) [reply index.html:3]: ok

**Review by Chikopes — APPROVED** [review]
IT's OK all.

**Pull request merged**